webapp/channels/src/components/post_view/post_add_channel_member/post_add_channel_member.tsx

```diff
@@ -152,33 +151,50 @@ export default class PostAddChannelMember extends React.PureComponent<Props, Sta
             return null;
         }
 
-        let linkId;
-        let linkText;
+        let link;
         if (channelType === Constants.PRIVATE_CHANNEL) {
-            linkId = t('post_body.check_for_out_of_channel_mentions.link.private');
-            linkText = 'add them to this private channel';
+            link = (
+                <FormattedMessage
+                    id='post_body.check_for_out_of_channel_mentions.link.private'
+                    defaultMessage='add them to this private channel'
```

Comment:
Some of these should be rewritten so that we aren't concatenating strings together because sentence order can be different in other languages, but that's another future task